addons/website_quote/static/src/css/website_quotation.css

```diff
@@ -42,7 +42,10 @@
   padding-left: 30px;
   font-size: 90%;
 }
-
+.o_website_quote .quote-state{
+    font-size:23.4px;
```

Comment:
23.4 ? seems very precise, where does that number comes from ? Try to use em instead of px also for fonts.